drivers/hud_apr/app/controllers/hud_apr/cell_details_concern.rb

```diff
@@ -0,0 +1,69 @@
+###
+# Copyright 2016 - 2021 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module HudApr::CellDetailsConcern
+  extend ActiveSupport::Concern
```

Comment:
This should be a model concern rather than a controller concern.  We prefer fat models over fat controllers and fat controllers over fat views.  I _think_ these can be mixed into the questions, so you can just ask a particular question for `question_fields` and get back the appropriate hash of `field_name => 'header text'`.  The base class can default to returning all columns so we fail over to the current setup.

Reply:
✅